doc/install.asciidoc

```diff
@@ -51,11 +51,23 @@ Download the https://packages.debian.org/sid/all/qutebrowser/download[qutebrowse
 https://packages.debian.org/sid/all/python3-pypeg2/download[PyPEG2]
 package from the Debian repositories.
 
+----
+# wget http://ftp.debian.org/debian/pool/main/p/pypeg2/python3-pypeg2_2.15.2-1_all.deb 
```

Comment:
I'm a bit worried we'll have to keep updating this link every release. I wonder if there's a way to point to the latest version or patch this with the release script.

Reply:
Hm yeah - when I suggested this I thought there would be an easy way to link to the latest .deb, but I don't think there is. Even if we'd only update this on releases, it'd be cumbersome to release a `-2` package with some packaging related changes, so I'd propose to just remove this again. Sorry @fiete201! :smile:

Reply:
A close alternative is just linking to https://packages.debian.org/sid/all/qutebrowser/download and ansking to download it from a mirror :P

Reply:
Which is already done in the sentence above :wink: